### PR TITLE
[paperless] New chart

### PR DESCRIPTION
**Pull request by mr-onion-2**
My first contribution to this great project.

This relies on a small tweak to the common chart (#253)

Hope this is all OK. Happy to make any changes you suggest.

Thanks

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/paperless/values.yaml
Comment (onedr0p) [inline charts/paperless/values.yaml:7]: `args` should really not be a part of `image` but instead it's own parent.

File: charts/paperless/values.yaml
Comment (onedr0p) [inline charts/paperless/values.yaml:72]: No need to include the rest here and under... ingress false is just fine

File: charts/paperless/values.yaml
Comment (onedr0p) [inline charts/paperless/values.yaml:15]: You can leave a default for this environment variable and remove this comment. The user can override it

**Review by onedr0p — COMMENTED** [review]
Thanks for the new chart. only a few things i would change

File: charts/paperless/values.yaml
Comment (bjw-s) [reply charts/paperless/values.yaml:72]: Since `ingress` is already disabled in the common library by default it's not necessary to include here

File: charts/paperless/Chart.yaml
Comment (bjw-s) [inline charts/paperless/Chart.yaml:13]: Please add the maintainers info. Feel free to add the default maintainers if you don't wish to add yourself.

File: charts/paperless/Chart.yaml
Comment (bjw-s) [inline charts/paperless/Chart.yaml:5]: First published version should probably be 1.0.0 🙂

File: charts/paperless/README.md
Comment (bjw-s) [inline charts/paperless/README.md:5]:
```suggestion
**This chart is not maintained by the upstream project and any issues with the chart should be raised [here](https://github.com/k8s-at-home/charts/issues/new/choose)**
```

File: charts/paperless/Chart.yaml
Comment (bjw-s) [inline charts/paperless/Chart.yaml:18]:
```suggestion
    version: 1.6.3
```
The ^ character breaks our RenovateBot integration 🙁. Don't forget to update to the actual `common` version once it's published.

File: charts/paperless/values.yaml
Comment (bjw-s) [inline charts/paperless/values.yaml:31]: We tend to set the default value for `enabled` to `false`. This makes sure that our CI does not fail, and that the chart user has to explicitly set persistence preferences. 

If they are absolutely required for the application to run, make sure that you provide a `ci/ct-values.yaml` that contains an `emptyDir` for these volumes.

File: charts/paperless/values.yaml
Comment (bjw-s) [inline charts/paperless/values.yaml:35]: This can be left out by default. The common library will handle this if the chart user provides these keys.

File: charts/paperless/values.yaml
Comment (bjw-s) [inline charts/paperless/values.yaml:37]: See comments above 🙂

File: charts/paperless/values.yaml
Comment (bjw-s) [inline charts/paperless/values.yaml:43]: See above 🙂

File: charts/paperless/values.yaml
Comment (bjw-s) [inline charts/paperless/values.yaml:49]: See above 🙂

**Review by bjw-s — CHANGES_REQUESTED** [review]

File: charts/paperless/values.yaml
Comment (mr-onion-2) [reply charts/paperless/values.yaml:7]: Cheers, done

File: charts/paperless/values.yaml
Comment (mr-onion-2) [reply charts/paperless/values.yaml:72]: Cheers done

**Comment by mr-onion-2** [comment]
Thanks for the feedback. I hope I've captured everything in the latest commit

File: charts/paperless/values.yaml
Comment (onedr0p) [inline charts/paperless/values.yaml:45]: You may want to specify a tag here, even if it is `latest`

```image: thepaperlessproject/paperless:latest```

**Comment by onedr0p** [comment]
@mr-onion-2 jump on our discord when you have a sec in #charts, I think we might need to discuss how to make this pass CI with volumes and the additionalContainer.

**Review by bjw-s — APPROVED** [review]
LGTM 🎉 Gonna let @onedr0p weigh in too before I press the merge button 🙂

**Comment by onedr0p** [comment]
Looks good to me too!

**Pull request merged**
